lib/fizzy/instantiate.hpp

```diff
@@ -137,6 +137,22 @@ struct ImportedFunction
 std::vector<ExternalFunction> resolve_imported_functions(
     const Module& module, const std::vector<ImportedFunction>& imported_functions);
 
+// Global that should be used by instantiate as import, identified by module and global name.
+struct ImportedGlobal
+{
+    std::string module;
+    std::string name;
+    Value* value = nullptr;
+    ValType type = ValType::i32;
+    bool is_mutable = false;
+};
+
+// Create vector of ExternalGlobals ready to be passed to instantiate.
+// imported_globals may be in any order, but must contain globals for all of the imported global
```

Comment:
Just realised these comments are no doxygen, neither for `resolve_imported_functions`. If we want to doxygenize them, lets do it in a new PR.